firmware/controllers/serial/serial.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file	serial.h
+ *
+ * @date Mar 19, 2020
+ * @author Konstantin Smola, (c) 2020
+ */
+
+#pragma once
+
+#include "hal.h"
+
+#include "periodic_thread_controller.h"
+
+#define TIMEREAD ((sysinterval_t)chTimeMS2I(10)) //10ms timeout
+
+#define SERBUFFLEN 64
+#define AUX_SERIAL_PORT (&SD6)
```

Comment:
nope, that would not go. we cannot have HW settings in weird places all over.

Reply:
I'll move it to efifeatures.h